perun-base/src/main/resources/perun-base.xml

```diff
@@ -45,6 +45,8 @@
 		<property name="attributesForUpdateIdP" value="#{'${perun.attributesForUpdate.idp}'.split('\s*,\s*')}"/>
 		<property name="attributesForUpdateX509" value="#{'${perun.attributesForUpdate.x509}'.split('\s*,\s*')}"/>
 		<property name="oidcIssuers" value="#{'${perun.oidc.issuers}'.split('\s*,\s*')}"/>
+		<property name="allowedCorsDomains" value="#{'${perun.allowedCorsDomains}'.split('\s*,\s*')}" />
+		<property name="csrfEnabled" value="${perun.csrfEnabled}" />
```

Comment:
I do think that the purpose of this flag is to prevent CSRF attacks, not to enable them :-) Thus the name should be the opposite, something like csrfProtectionEnabled.

Reply:
Renamed to csrfProtectionEnabled.